admin/nodes/views.py

```diff
@@ -497,6 +499,72 @@ def get_context_data(self, **kwargs):
         context['resource_type'] = 'node'
         return context
 
+class NodeModifyStorageUsage(TemplateView):
+    template_name = 'nodes/modify_storage_caps.html'
+
+    def get_object(self, queryset=None):
+        node = Node.load(self.kwargs.get('guid')) or Registration.load(self.kwargs.get('guid'))
+        return {
+            'id': node.id,
+        }
+
+    def get_context_data(self, **kwargs):
+        context = super(NodeModifyStorageUsage, self).get_context_data(**kwargs)
```

Comment:
When using `super` in py3 you don't have to name check the superclass anymore, so this can be `super().get_context_data(**kwargs)` same this the other `super` calls. Makes it easier to refactor.

Reply:
👍